src/site/content/en/metrics/inp/index.md

```diff
@@ -140,7 +144,7 @@ INP is more than about first impressions. By sampling all interactions, responsi
 It's possible that a page can return no INP value. This can happen for a number of reasons:
 
 - The page was loaded, but the user never clicked, tapped, or pressed a key on their keyboard.
-- The page was loaded, but the user interacted with the page using gestures that did not involve clicking, tapping, or using the keyboard. For example, scrolling or hovering over elements does not factor into how INP is calculated.
+- The page was loaded, but the user interacted with the page using gestures that did not involve clicking, tapping, or using the keyboard. For example, scrolling or hovering over elements does not factor into how INP is calculated (unless the space bar was used to scroll the page).
```

Comment:
Given the above clarity, I think we can remove this.  Scroll is never measured, its always just keyboard.  The fact that some keyboard actions scroll by default I think is addressed in the detail above.